core/templates/pages/about-page/about-page.module.ts

```diff
@@ -33,13 +33,18 @@ import { HttpClientModule } from '@angular/common/http';
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { RequestInterceptor } from 'services/request-interceptor.service';
 import { SharedComponentsModule } from 'components/shared-component.module';
-// This component is needed to force-bootstrap Angular at the beginning of the
-// app.
-@Component({
-  selector: 'service-bootstrap',
-  template: ''
-})
-export class ServiceBootstrapComponent {}
+
+import { OppiaAngularRootComponent } from
+  'components/oppia-angular-root.component';
+import { CamelCaseToHyphensPipe } from
```

Comment:
Should we really do this in all modules -- adding all needed pipes in each module?

Don't you think a better way would be to add all the pipes in shared-module.component.ts (it is already added to the imports of each module)

Reply:
Shared module didn't work. I will try to add an injectable decorator to these pipes and see if that helps me to get rid of these.

Reply:
Why shared-module didn't work? That's the correct way to add things IMO.

Reply:
Actually, in the case of Dependency Injection, devs usually don't "provide" di in shared module. If something is a DI, it goes straight into app.module.ts. But we don't have one right now.